common/terms.html

```diff
@@ -91,6 +96,16 @@
     <dd>
       <p>Content provided for information purposes and not required for conformance. Content required for conformance is referred to as <a>normative</a>.</p>
     </dd>
+    <dt><dfn>Identifies</dfn></dt>
+    <dd>
+      <p>Used in an attribute description to denote that the value <a href="#typemapping">type</a> is an IDREF (identifying a single element) or IDREFS (identifiying one or more elements).</p>
+      <p>Related Terms: <a>Defines</a>, <a>Indicates</a></p>
+    </dd>
+    <dt><dfn>Indicates</dfn></dt>
+    <dd>
+      <p>Used in an attribute description to denote that the value <a href="#typemapping">type</a> is a Boolean (true/false), an NMTOKEN (a single named token), or NMTOKENS (a list of one or more named tokens).</p>
```

Comment:
Yes, I thought about that and didn’t want to enumerate ever type, but if your or JamesN feels strongly about it, I will change it.

Reply:
I think it's only the 2 extra types: true/false/undefined and tristate. Admittedly, those boolean-token-fusion types are a bit weird to describe. Maybe it would help to link to the characteristics types, something like:
```suggestion
      <p>Used in an attribute description to denote that the value <a href="#propcharacteristic_value">type</a> is <a href="#valuetype_true-false">true/false</a>, <a href="#valuetype_true-false-undefined">true/false/undefined</a>, <a href="#valuetype_tristate">tristate</a>, a single named <a href="#valuetype_token">token</a>, or a <a href="#valuetype_token_list">token list</a>.</p>
```

Interested to see what @jnurthen says.

Reply:
To be honest I don't really care that much. But my 2¢ is that as this is the ARIA spec we should probably use the ARIA terms. We can then include a link to https://w3c.github.io/aria/#typemapping which contains mappings to HTML and XML definitions for each term. 
but I'm happy to go another direction.

Reply:
```suggestion
      <p>Used in an attribute description to denote that the value <a href="#propcharacteristic_value">type</a> is any of the token or token-like values, including the Boolean-like <a href="#valuetype_true-false">true/false</a> tokens, <a href="#valuetype_true-false-undefined">true/false/undefined</a> token, <a href="#valuetype_tristate">tristate (true/false/mixed)</a> token, a single named <a href="#valuetype_token">token</a>, or a <a href="#valuetype_token_list">token list</a>.</p>
```